spring-cloud-common-security-config/spring-cloud-common-security-config-web/src/main/java/org/springframework/cloud/common/security/support/SecurityConfigUtils.java

```diff
@@ -73,4 +75,23 @@ public static ExpressionUrlAuthorizationConfigurer<HttpSecurity>.ExpressionInter
 		}
 		return security;
 	}
+
+	public static void configureSimpleSecurity2(
```

Comment:
Can we rename this to configureSimpleSecurity and remove the unused version?

Reply:
That was my oversight.